### PR TITLE
warn if uploaded pd file is empty

### PR DESCRIPTION
**Pull request by fanjinfei**

### CODE REVIEW

**Comment by fanjinfei** [comment]
ready for review

File: ckanext/recombinant/controller.py [hunk 1]
```diff
@@ -294,3 +298,5 @@ def _process_upload_file(lc, dataset, upload_file, geno):
             raise BadExcelData(
                 _(u"Error while importing data: {0}").format(
                     pgerror))
+    if total_records==0:
+        raise BadExcelData(_("The template updated is empty"))
```
Comment (wardi) [inline ckanext/recombinant/controller.py]: `if not total_records:` is more common in python than `==0`

Can we change this text to `The template uploaded is empty`? `updated` sounds pretty awkward to me

File: ckanext/recombinant/controller.py [hunk 2]
```diff
@@ -277,6 +278,9 @@ def _process_upload_file(lc, dataset, upload_file, geno):
 
         records = get_records(rows, chromo['fields'])
         method = 'upsert' if chromo.get('datastore_primary_key') else 'insert'
+        total_records += len(records)
+        if len(records)==0:
```
Comment (wardi) [inline ckanext/recombinant/controller.py]: `if not records:`

File: ckanext/recombinant/controller.py (hunk 1 above)
Comment (fanjinfei) [inline ckanext/recombinant/controller.py]: I will ask Claudie Larouche for approval of the En/Fr text.

**Pull request merged**